Observing-Interface-Setup/production environment_package_management.rst

```diff
@@ -0,0 +1,182 @@
+.. Do not remove the comments that describe each section. They are included to provide guidance to contributors.
+.. Do not remove other content provided in the templates, such as a section. Instead, comment out the content and include comments to explain the situation. For example:
+	- If a section within the template is not needed, comment out the section title and label reference. Include a comment explaining why this is not required.
+    - If a file cannot include a title (surrounded by ampersands (#)), comment out the title from the template and include a comment explaining why this is implemented (in addition to applying the ``title`` directive).
+
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: *Patrick Ingraham*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *Tiago Ribeiro*
+
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. _Observing-Interface-Production-Environment-Package-Management:
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+.. _Production-Environment-Package-Management:
+
+#########################################
+Production Environment Package Management
+#########################################
+
+Introduction
+^^^^^^^^^^^^
+
+.. This section should provide a brief, top-level description of the procedure's purpose and utilization. Consider including the expected user and when the procedure will be performed.
+
+This page explains how package management in controlled on production environments, specifically the summit.
+Nominally, the CSC versions and supporting software packages deployed on the summit are managed by the `cycle build <https://ts-cycle-build.lsst.io/>`_.
+However, during commissioning it will be required to deploy new versions for testing, and/or deploy on-the-fly hotfixes that won't yet have been included as part of the cycle build.
+These fixes must be simultaneously available to the ScriptQueue CSC as well as the observer's notebook environments.
+Most importantly, when testing new software on the summit, which may not be entirely stable, it is critical to have a mechanism to immediately roll back all packages to a designated stable version.
+We call this the **base environment**.
+Note that this version may include bug fixes and/or new functionalities that are not included in the previously released cycle build.
```

Comment:
I think, ideally, the "base environment" would match a revision of the cycle build. This means we have a unified way to control what we deploy at the summit. So the procedure to update the "base environment" would basically be to create a new revision. Now, this doesn't mean we have to build a new version of the SQ and nublado to get it deployed.

Reply:
"basically create a new revision" --> What you really mean is update a local version of `cycle.env`, then have some sort of process that matches the NFS mounted packages to that file, yes?
I think for clarity we really should refer to `cycle.env` as something else though... 
Also, we should strip out all the packages (e.g. CSCs) that would be no-ops as they'll just add confusion.